src/app/modules/item/pages/item-log-view/item-log-view.component.ts

```diff
@@ -26,12 +27,8 @@ export class ItemLogViewComponent implements OnChanges {
   @Input() itemData?: ItemData;
   @Input() isWatchingGroup = false;
 
-  private logColumns?: Column[];
-
-  private readonly id$ = new ReplaySubject<string>(1);
-  readonly state$ = this.id$.pipe(
-    distinct(),
-    switchMap(id => this.getData$(id)),
+  private readonly item$ = new ReplaySubject<Item>(1);
```

Comment:
And always complete your subject !!!! Need to be completed on destroy.

Reply:
Fixed